sdk/webpubsub/Azure.Messaging.WebPubSub/src/Generated/WebPubSubServiceClient.cs

```diff
@@ -345,6 +475,22 @@ internal virtual Response ConnectionExistsImpl(string connectionId, RequestConte
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="connectionId"/> is null. </exception>
         /// <exception cref="ArgumentException"> <paramref name="connectionId"/> is an empty string, and was expected to be non-empty. </exception>
+        /// <example>
+        /// This sample shows how to call CloseConnectionAsync with required parameters and properties.
+        /// <code>
+        /// var credential = new DefaultAzureCredential();
+        /// var endpoint = new Uri("https://my-account-name.azure.com");
+        /// var client = new WebPubSubServiceClient(endpoint, credential);
+        /// Response response = await client.CloseConnectionAsync("connectionId");
+        /// </code>
+        /// This sample shows how to call CloseConnectionAsync with all parameters and properties.
+        /// <code>
+        /// var credential = new DefaultAzureCredential();
+        /// var endpoint = new Uri("https://my-account-name.azure.com");
+        /// var client = new WebPubSubServiceClient(endpoint, credential);
+        /// Response response = await client.CloseConnectionAsync("connectionId", null);
```

Comment:
Why are we passing `null` here?

Reply:
Previously I used the `DefaultValue`. I've improved the logic to not use `null ` default values.